docs/tempo/website/metrics-generator/service_graphs.md

```diff
@@ -21,46 +21,49 @@ and the connections and dependencies between its components:
   Distributed systems change very frequently,
   and service graphs offer a way of seeing how these systems have evolved over time.
 
-<p align="center"><img src="../service-graphs.png" alt="Service graphs example"></p>
+<p align="center"><img src="../grafana-service-graphs-panel.png" alt="Service graphs example"></p>
 
 ## How they work
 
 The metrics-generator will process traces and generate service graphs in the form of prometheus metrics.
 
 Service graphs work by inspecting traces and looking for spans with parent-children relationship that represent a request.
-Additionally, spans need to contain the tag [`span.kind`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spankind),
-with value `CLIENT` for the parent span (i.e. service that started the request) and `SERVER` for the children span (i.e. service that received the request).
+The processor uses the [OpenTelemetry semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md) to detect a myriad of requests.
```

Comment:
> to detect a myriad of requests

do you mean to _inspect_ a myriad of requests?

Reply:
I used detect here because the processor focuses on discovering the requests and counting them. It doesn't really inspect them besides adding attributes to the metrics.